src/snowflake/cli/plugins/object/stage/commands.py

```diff
@@ -121,3 +123,27 @@ def stage_diff(
     """
     diff: DiffResult = stage_diff(Path(folder_name), stage_name)
     return ObjectResult(str(diff))
+
+
+@app.command("execute", requires_connection=True)
+def execute(
+    stage_path: str = typer.Argument(
+        None,
+        help="Execute immediate all files from the path. Files can be filtered with glob like pattern, e.g. `@stage/**/*.sql`.",
+    ),
+    on_error: OnErrorType = typer.Option(
+        OnErrorType.BREAK.value,
+        "--on-error",
+        help="What to do when an error occurs. Defaults to break.",
+    ),
+    parameters: Optional[List[str]] = typer.Option(
+        None,
+        help='Parameters for the template. For example: `--parameters "<key>=<value>"`, string values should be in single quotes.',
```

Comment:
Currently usage of this is `--parameters "key1=value1 --parameters "key2=value2"...`. I'd change the name to `parameter` (singular seems more intuitive for me)
Also - do we want to add some abbreviation (`--param` / `-p`)? Just not to write `--parameter` for absolute every value

Reply:
I unified variables with notebooks and snowsql, now is `-D`